packages/backend/src/graphql/workHour/mutations/workHourCreateMutationField.ts

```diff
@@ -9,14 +9,7 @@ builder.mutationField('workHourCreate', (t) =>
     args: {
       data: t.arg({ type: WorkHourInput }),
     },
-    authScopes: async (_source, { data: { taskId } }) => {
-      const task = await prisma.task.findUniqueOrThrow({
-        select: { projectId: true },
-        where: { id: taskId.toString() },
-      })
-
-      return { isProjectMember: task.projectId }
-    },
+    authScopes: (_source, { data: { taskId } }) => ({ isTaskAdmin: taskId.toString() }),
```

Comment:
a bit late in the review, what is a task Admin? This scope should be only about adding work hours, not editing tasks?

Reply:
`isTaskAdmin` checks, if the session user has the role admin in the project of the task. But I agree, the name is misleading.
@moncapitaine What do you think about `isProjectAdminOfTask` and `isProjectMemberOfTask`, or do you have a better name for this scope?

---

Also, every member should create work hours, the logic here is wrong. I will change this.